charts/prometheus/values.yaml

```diff
@@ -611,100 +638,91 @@ prometheus-rules:
             expr: sum by (datname, label_app_kubernetes_io_instance) (rate(pg_stat_database_xact_commit{datname!~"^(postgres|template[0-9]*|default|cloudsqladmin|unknown)$"}[5m])) < 0.5
             for: "2m"
             description: "Transactions per second for database {{ $labels.datname }} in {{ $labels.label_app_kubernetes_io_instance }} service is below norm"
+          Postgres_is_down:
+            enabled: false
+            for: "5m"
+            description: "Postgres {{ $labels.pod }} in {{ $labels.namespace }} is down"
+            expr: pg_up == 0
+          Postgres_replica_is_down:
+            enabled: false
+            for: "5m"
+            description: "Postgres replica {{ $labels.label_app_kubernetes_io_instance }} in {{ $labels.namespace }} is down"
+            expr: sum by (label_app_kubernetes_io_instance, namespace) (pg_replication_is_replica) < 1
+
+          # Redis
+          Redis_masters:
```

Comment:
lets leave the previous alert name Redis_too_many_masters. Explicit is better than implicit © Zen of Python

Reply:
same for rabbitmq